apps/documentation/docs/getting-started/serverless.md

```diff
@@ -2,24 +2,286 @@
+    path: path.join(__dirname, '.webpack'),
+    filename: '[name].js',
+  },
+  // experiments: {
+  //   topLevelAwait: true,
+  // },
+  target: 'node',
+  externalsPresets: { node: true },
+  externals: [nodeExternals()],
+  optimization: {
+    minimize: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(tsx?)$/,
+        loader: 'babel-loader',
+        exclude: [
+          [
+            path.resolve(__dirname, 'node_modules'),
+            path.resolve(__dirname, '.serverless'),
+            path.resolve(__dirname, '.webpack'),
+          ],
+        ],
+        options: {
+          presets: [
+            ['@babel/preset-env', { targets: { node: 'current' } }],
+            '@babel/preset-typescript',
+          ],
+          plugins: [
+            'babel-plugin-transform-typescript-metadata',
+            ['@babel/plugin-proposal-decorators', { legacy: true }],
+            ['@babel/plugin-proposal-class-properties', { loose: true }],
+            'babel-plugin-parameter-decorator',
+            ['@babel/plugin-proposal-private-methods', { loose: true }],
+            ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
+          ],
+        },
+      },
+    ],
+  },
+  plugins: [new ForkTsCheckerWebpackPlugin()],
+};
+```
+
+### Configure Serverless YML file
+
+- Create the `serverless.yml` file with the following configuration:
+
+```yml
+service: hello
+
+custom:
+  webpack:
+    webpackConfig: webpack.config.js
+    includeModules:
+      forceExclude:
+        - aws-sdk
+    packager: npm
+    packagerOptions:
+      lockFile: '../../package-lock.json'
+  prune:
+    automatic: true
+    number: 3
+
+plugins:
+  - serverless-offline
+  - serverless-webpack
+  - serverless-iam-roles-per-function
+  - serverless-create-global-dynamodb-table
+  - serverless-prune-plugin
+
+provider:
+  name: aws
+  region: us-east-1
+  runtime: nodejs14.x
+  memorySize: 256
+  httpApi:
+    payload: '2.0'
+    cors: true
+```
+
+### Create a Serverless function
+
+- Create file `src/handler.ts`, in which will contain the serverless function.
+- An example of the `src/handler.ts` will be as follow:
+
+```ts
+export const sendEmailHandler = async (event: APIGatewayProxyEvent) => {
+  const eventBody = JSON.parse(event.body || '{}') as SendEmailConfig;
+  if (!validateRequest(eventBody)) {
+    return {
+      msg: 'Missing fields',
+    };
+  }
+
+  const sendEmailStatus = await sendEmail(eventBody);
+  return {
+    msg: sendEmailStatus ? 'Successfully sent email' : 'Unsucessfully sent email',
+  };
+};
+```
+
+- Add the following into `serverless.yml`:
+
+```yml
+functions:
+  sendgrid:
+    handler: src/handler.sendEmailHandler
+    events:
+      - httpApi:
+          # This is endpoint for function
+          path: /email
+          # This is HTTP method
+          method: post
+```
+
+### Run the function locally
+
+- To run an emulator of the function run the following command:
```

Comment:
clarify how to run using turbo as well `npx turbo run sls:offline --scope=@acmutd/<project>` from root dir